### DISCUSSION TITLE
bridge mode

### DISCUSSION THREAD

**Discussion by ish-org** [pgj/freebsd-wifibox#52] [Q&A]
I want use wifibox with bridge mode.
Is there any way to bridge between eth0 and wlan0?
Because I want to make lagg with re0 and wifibox0.

**Comment by pgj** [comment]
Hi @ish-org, as far as I have learned, it is not possible to bridge between `eth0` and `wlan0` (inside the VM, I believe that is what you meant).  In #18 we had some discussion on this topic, although the focus has recently moved to firewall configuration due to lack of true bridging.

**Comment by ish-org** [comment]
Thank you for reply.
It is popular to make a WiFi router using a Raspberry Pi, and in that case bridge-utils is used to achieve bridge mode, so I wondered if it would be possible to do the same on a VM. There seems to be something else called 'Using Netgraph for FreeBSD's bhyve Networking'(https://klarasystems.com/articles/using-netgraph-for-freebsds-bhyve-networking/).

**Comment by pgj** [comment]
Thanks for calling my attention to the Netgraph-enabled `bhyve`.  I use FreeBSD 12 as the primary developer system, maybe that is why I have not notice this feature yet.  But I could explore this approach and see how this works.  In case of success, it might be worth to port it back to FreeBSD 12 (via `bhyve+`) if possible.
